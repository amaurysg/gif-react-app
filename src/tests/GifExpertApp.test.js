import React from 'react'
import { shallow } from "enzyme"
import GifExpertApp from "../GifExpertApp"

describe('Test component <GifExpertApp/>', () => {

  
  test('Render component Ok', () => {
    const wrapper = shallow(<GifExpertApp/>)
    expect(wrapper).toMatchSnapshot()
  })
  test('should ', () => {
    const categories =['Dragon Ball','Naruto']
    const wrapper = shallow(<GifExpertApp  defaultCategory={categories} />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.find('GifGrid').length).toBe(categories.length)
    
  })
  
  
})
