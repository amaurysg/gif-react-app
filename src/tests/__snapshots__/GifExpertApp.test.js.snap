// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component <GifExpertApp/> Render component Ok 1`] = `
<div>
  <h2
    className="animate__animated animate__slow 5s animate__flash animate__infinite	infinite "
  >
    Gif Expert
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <ol />
</div>
`;

exports[`Test component <GifExpertApp/> should  1`] = `
<div>
  <h2
    className="animate__animated animate__slow 5s animate__flash animate__infinite	infinite "
  >
    Gif Expert
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <ol>
    <GifGrid
      category="Dragon Ball"
      key="Dragon Ball"
    />
    <GifGrid
      category="Naruto"
      key="Naruto"
    />
  </ol>
</div>
`;
